### PR TITLE
Video Gallery Drawer Menu

### PR DESCRIPTION
**Pull request by anjulgarg**
# What
Video Gallery Drawer Menu
Screenshot 2022-12-19 at 3 31 49 PM

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/review/stable/react-components.api.md [hunk 1]
```diff
@@ -1188,6 +1188,7 @@ export const _RemoteVideoTile: React_2.MemoExoticComponent<(props: {
     strings?: VideoGalleryStrings | undefined;
     participantState?: ParticipantState | undefined;
     showRemoteVideoTileContextualMenu?: boolean | undefined;
+    drawerMenuHostId?: string | undefined;
```
Comment (edwardlee-msft) [inline packages/react-components/review/stable/react-components.api.md:1191]: Should this prop be in stable as well?

File: packages/react-components/src/components/VideoGallery.tsx [hunk 2]
```diff
@@ -363,6 +369,7 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
       onRenderAvatar,
       showMuteIndicator,
       strings,
+      /* @conditional-compile-remove(pinned-participants) */ drawerMenuHostId,
```
Comment (edwardlee-msft) [inline packages/react-components/src/components/VideoGallery.tsx:372]: Why is this conditionally compiled while it's usage is not?

File: packages/react-components/src/components/RemoteVideoTile.tsx [hunk 3]
```diff
@@ -131,24 +136,39 @@ export const _RemoteVideoTile = React.memo(
...
-        personaMinSize={props.personaMinSize}
-        showLabel={props.showLabel}
-        /* @conditional-compile-remove(one-to-n-calling) */
-        /* @conditional-compile-remove(PSTN-calls) */
-        participantState={participantState}
-        {...videoTileContextualMenuProps}
-        /* @conditional-compile-remove(pinned-participants) */
-        isPinned={props.isPinned}
-      />
+      <Stack style={{ height: '100%', width: '100%', position: 'relative' }}>
```
Comment (alkwa-msft) [inline packages/react-components/src/components/RemoteVideoTile.tsx:139]: can we make this an inline style?

File: packages/react-components/review/stable/react-components.api.md (hunk 1 above)
Comment (anjulgarg) [reply packages/react-components/review/stable/react-components.api.md:1191]: Yes. It's an internal component.
If it was being exported by communication-react.api.md, that would be a problem

File: packages/react-components/src/components/VideoGallery.tsx (hunk 2 above)
Comment (anjulgarg) [reply packages/react-components/src/components/VideoGallery.tsx:372]: Good point. It's unneeded. Lemme remove it :)

**Review by alkwa-msft — APPROVED** [review]

File: packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx [hunk 4]
```diff
@@ -47,6 +47,7 @@ export interface MediaGalleryProps {
   onRenderAvatar?: OnRenderAvatarCallback;
   onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
   isMobile?: boolean;
+  drawerMenuHostId?: string;
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx:50]: this prop is added here but looks unused?

**Review by JamesBurnside — APPROVED** [review]

**Review by edwardlee-msft — APPROVED** [review]

**Pull request merged**